packages/core/src/CommandManager.ts

```diff
@@ -120,10 +120,6 @@ export class CommandManager {
     const { rawCommands, editor, state } = this
     const { view } = editor
 
-    if (state.storedMarks) {
```

Comment:
https://prosemirror.net/docs/ref/#state.EditorState.storedMarks

From the context of the file it seems like this is used when chains are created / can commands are created.

From the Prosemirror doc it seems like this is used to save marks between transactions and reapply them after a transaction.

I could guess there are some edge cases where this could happen. Do you think your commit would break this behavior? Maybe it's not even that bad since this is only relevant for chain commands anyway?